$NetBSD: patch-ad,v 1.1 2009/02/07 18:38:06 jakllsch Exp $

Allow use of IPv6/getaddrinfo even if AI_ADDRCONFIG extension
isn't present.

--- src/resolver.c.orig	2009-01-09 13:03:04.000000000 +0000
+++ src/resolver.c
@@ -49,8 +49,9 @@
 #endif
 
 #ifndef MPD_NO_GAI
-#  ifdef AI_ADDRCONFIG
-#    define MPD_HAVE_GAI
+#  define MPD_HAVE_GAI
+#  ifndef AI_ADDRCONFIG
+#    define AI_ADDRCONFIG 0
 #  endif
 #endif
 
